packages/dashboard-frontend/src/store/ServerConfig/helpers.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export function isSourceAllowed(allowedSourceUrls: string[], url: string): boolean {
```

Comment:
Could you move the empty array check here, please?

this one https://github.com/eclipse-che/che-dashboard/pull/1192/files#diff-d201f5db010e159f1e4924acc8c0e5b0741f40c4f60aed6440327ddb250318d4R275-R277